ci-operator/step-registry/openshift/e2e/gcp/custom-dns/private/openshift-e2e-gcp-custom-dns-private-workflow.yaml

```diff
@@ -0,0 +1,25 @@
+workflow:
+  as: openshift-e2e-gcp-custom-dns-private
+  steps: 
+    pre:
+    - ref: gcp-provision-minimal-permission
+    - ref: gcp-provision-vpc
+    - ref: ignition-bastionhost
+    - ref: gcp-provision-bastionhost
+    - chain: ipi-conf-gcp
+    - ref: ipi-install-rbac
+    - ref: openshift-cluster-bot-rbac
+    - ref: ipi-install-hosted-loki
+    - ref: ipi-install-install-in-bastion
+    - ref: gcp-provision-private-custom-dns
+    - ref: bastion-dnsmasq
+    - ref: proxy-config-generate
+    - chain: cucushift-installer-check
+    - ref: cucushift-installer-check-gcp-private
+    - ref: cucushift-installer-check-user-provisioned-dns
+    test:
+    - chain: cucushift-installer-check-cluster-health
```

Comment:
```suggestion
    - chain: openshift-e2e-test
```

Reply:
@yunjiang29 the `openshift-e2e-test` can't pass on private cluster, so we also dropped the test from the common private cluster testing, like: https://github.com/openshift/release/blob/master/ci-operator/config/openshift/installer/openshift-installer-main.yaml#L1317

Reply:
switched the test to public cluster, so added `openshift-e2e-test` in the workflow